cst/plugins/eu.esdihumboldt.cst.functions.numeric/src/eu/esdihumboldt/cst/functions/numeric/sequentialid/SequentialID.java

```diff
@@ -86,11 +86,17 @@ protected Object evaluate(String transformationIdentifier, TransformationEngine
 				id = seqValue + 1;
 			}
 			else {
+				if (Integer.parseInt(startValue) == 0) {
```

Comment:
`if (Integer.parseInt(startValue) == 0) {` will throw a NumberFormatException if it is not able to parse the string value to Integer. Also, as it is not in a try catch block your code will explode.
Although, we know that we have done the validations in the UI to make sure that no string value is transferred to this function, I would still use `id`, i.e. `if(id == 0)` for this validation in the `try catch` block below.